README.md

```diff
@@ -40,7 +44,12 @@ go get github.com/adshao/go-binance/v1
 
 ```golang
 import (
+    // for spot and other interfaces contained in https://binance-docs.github.io/apidocs/spot/en/#change-log
     "github.com/adshao/go-binance/v2"
+    
+    "github.com/adshao/go-binance/v2/futures" // optional package
+    "github.com/adshao/go-binance/v2/delivery" // optional package
+    "github.com/adshao/go-binance/v2/eoptions" // optional package
```

Comment:
To ensure compatibility, it's better to use `options` as the package name instead of `eoptions`. We can rename `options` to `eoptions` when Binance introduces American options.